cf/aggregate.py

```diff
@@ -1340,16 +1338,38 @@ def aggregate(fields,
         fields: `FieldList` or sequence of `Field`
             The field constructs to aggregated.
 
-        verbose: `int`, optional
-            Print information about the aggregation process. By
-            default *info* is 0 and no information is displayed. If
-            *info* is 1 or more then display information on which
-            fields are unaggregatable, and why. If *info* is 2 or more
-            then also display the structural signatures of the fields
-            and, when there is more than one field construct with the
-            same structural signature, their canonical first and last
-            coordinate values. If *info* is 3 or more then display the
-            field construct's complete aggregation metadata.
+        verbose: `int` or `None`, optional
+            If an integer from `0` to `3`, corresponding to increasing
```

Comment:
Hi - minor point - elsewhere we use double backticks on Python instances (``1``, ``'foobar'``, etc) and only use single backticks on classes/functions/etc that can link to docs (`None`, `int`, etc.)

Reply:
Thanks, noted (for this PR but also for future work, as consistency is important!). I'll amend the new docstring captions accordingly. I did copy them directly from cfdm, so I should amend the cfdm ones similarly, once I have this PR ready to go.

Reply:
Addressed in f9a4ce3dac8086c02b9ac8db8ef17f759b43aff8, cheers.